src/stream/KeyExchange.js

```diff
@@ -127,7 +146,7 @@ function GroupKeyStore({ groupKeys = new Map() }) {
         },
         clear() {
             currentGroupKeyId = undefined
-            nextGroupKey = undefined
+            nextGroupKeys.length = 0
```

Comment:
Could be simply `nextGroupKeys = []`? Also at line 167.

Reply:
I mean, sure, either way, not sure it makes a lot of difference, I was just working with it being a `const` in `const nextGroupKeys = []`.